### PR TITLE
Homework 1

### PR DESCRIPTION
**Pull request by Marimeiko**

### CODE REVIEW

File: module1/homework/calculate/calculate.hpp [hunk 1]
```diff
@@ -1,7 +1,27 @@
...
+        result = first - second;
+        std::cout << result;
+    }
+    else if (command == "divide") {
+        result = first / second;
+        std::cout << result;
+    }
+    else {
+        std::cout << "Invalid data";
+    }
```
Comment (piezol) [inline module1/homework/calculate/calculate.hpp:25]: Function 'calculate' should not have side effects (ie we should not use std::cout).

File: module1/homework/fibonacci/fibonacci.hpp [hunk 2]
```diff
@@ -1,11 +1,41 @@
...
-    // TODO: Your implementation goes here
-    return 0;
+
+    int first = 1;
+    int second = 1;
+    int result;
+
+    if (sequence <=3){
+        return 1;
+    }
```
Comment (piezol) [inline module1/homework/fibonacci/fibonacci.hpp:12]: fibonacci(3) is not equal to 1.

File: module1/homework/fibonacci/fibonacci.hpp [hunk 3]
```diff
@@ -1,11 +1,41 @@
...
 int fibonacci_recursive(int sequence) {
-    // TODO: Your implementation goes here
+
+    int first = 1;
+    int second = 1;
+    int result;
+
+    if (sequence <=3){
+        return 1;
+    }
```
Comment (piezol) [inline module1/homework/fibonacci/fibonacci.hpp:30]: Same thing.

File: module1/homework/fibonacci/fibonacci.hpp [hunk 4]
```diff
@@ -1,11 +1,41 @@
...
+
+    for (int i = 3; i <= sequence; i++) {
+        result=first+second;
+        second=first;
+        first=result;
+        std::cout << result << std::endl;
+    }
```
Comment (piezol) [inline module1/homework/fibonacci/fibonacci.hpp:37]: It isn't really recursive, is it?

File: module1/homework/calculate/calculate.hpp (hunk 1 above)
Comment (DamiXioR) [reply module1/homework/calculate/calculate.hpp:25]: you should just return result

File: module1/homework/fibonacci/fibonacci.hpp (hunk 2 above)
Comment (DamiXioR) [reply module1/homework/fibonacci/fibonacci.hpp:12]: it's good practice initialize created variable etc with 0

File: module1/homework/fibonacci/fibonacci.hpp (hunk 3 above)
Comment (DamiXioR) [reply module1/homework/fibonacci/fibonacci.hpp:30]: the same, please initialize variable

**Pull request closed without merging**